examples/door-lock-app/door-lock-common/door-lock-app.zap

```diff
@@ -6876,8 +6940,8 @@
               "storageOption": "RAM",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "0x103",
-              "reportable": 0,
+              "defaultValue": "0x113",
```

Comment:
Right, so this is claiming the SCH feature, but not adding all the things it requires....

Reply:
Yeah, I felt a little uncomfortable to do this but the door lock cluster is still in development and door lock group decided not to prioritize the holiday schedules for now.